lib/tests/modules/alias-with-priority-can-override.nix

```diff
@@ -41,7 +48,7 @@ with lib;
     # should override the next import.
     ( { config, lib, ... }:
       {
-        enableAlias = lib.mkForce false;
+        enableAlias = mkForce false;
```

Comment:
The `lib` here is from the set pattern right above, not from the top-level, so should this perhaps remain unchanged?

Reply:
I think it's clearer, since instead of two ways of referring to the same symbol, there's just one.

And, it turns out, it's also [more performant](https://github.com/NixOS/nix/issues/9592).

Reply:
In general, this would only be relevant when combining two versions of the module system.
On top of that, this is just a test, with no other version of  `lib` in sight.